### DISCUSSION TITLE
Github Action Quota reset problem

### DISCUSSION THREAD

**Discussion by vindia9** [community/community#25965] [Actions]
<p>Hi,</p>
<p>My Github Action quota shows “Included minutes quota resets in -2073 days”.<br>
I have been waiting for a month, it can not be reset, is that correct?</p>
<p>Thank you!</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/5/57a40a99102ca8cc7760a89c27c45d43c52c552d.png" data-download-href="/uploads/short-url/cviZOtQNpEXaN7i5vd0whqalXSR.png?dl=1" title="github" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/9072630/181094332-606608bf-2be7-4e61-929d-89285b3dd28d.png" alt="github" data-base62-sha1="cviZOtQNpEXaN7i5vd0whqalXSR" width="690" height="309" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/5/57a40a99102ca8cc7760a89c27c45d43c52c552d_2_690x309.png, https://aws1.discourse-cdn.com/github/optimized/2X/5/57a40a99102ca8cc7760a89c27c45d43c52c552d_2_1035x463.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/5/57a40a99102ca8cc7760a89c27c45d43c52c552d.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/5/57a40a99102ca8cc7760a89c27c45d43c52c552d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">github</span><span class="informations">1120×502 15.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by NicoNBW** [comment, marked as answer]
<p>Hi <a class="mention" href="/vindia9">@vindia9</a><br>
Regarding your question, i would recommend you contact <a href="https://support.github.com/contact" rel="nofollow noopener">GitHub Support</a> or <a href="https://premium.githubsupport.com/" rel="nofollow noopener">GitHub Premium Support </a></p>
